lib/rules/no-forbidden-elements.js

```diff
@@ -69,10 +81,20 @@ module.exports = class NoForbiddenElements extends Rule {
 
     throw new Error(errorMessage);
   }
+
+  isInHeadHbsFile(scope, node) {
+    return scope._filePath === 'app/templates/head.hbs' && node.tag === 'meta';
+  }
+
   visitor() {
     return {
-      ElementNode(node) {
+      ElementNode(node, path) {
+        if (hasHeadParent(path) || this.isInHeadHbsFile(this, node)) {
+          return;
+        }
```

Comment:
I think this is still an issue. This short circuit returns for `head.hbs` (or if the parent includes `head` element), but there are some forbidden element types that we should catch that wouldn't be caught by this logic.

Can you add the following "bad" test cases:


* With the `app/templates/head.hbs` filePath:

```hbs
<html></html>
```

* In any filePath other than `head.hbs`:

```hbs
<head>
  <html></html>
</head>
```

Reply:
ok, I'll work on it some more :)